src/components/Header/Header.tsx

```diff
@@ -1,11 +1,69 @@
-import React from 'react';
+import React, { FC } from 'react';
+import './header.scss';
+import NiceGadgetsLogo from '../../img/Nice Gadgets.png';
+import Favourite from '../../img/icons/favourite.svg';
+import Basket from '../../img/icons/shopping_bag.svg';
 import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
 
-export const Header = () => (
-  <nav>
-    <NavLink to="/">Home</NavLink>
-    <NavLink to="/phones">Phones</NavLink>
-    <NavLink to="/tablets">Tablets</NavLink>
-    <NavLink to="/accessories">Accessories</NavLink>
-  </nav>
-);
+export const Header: FC = () => {
+  return (
+    <div className="header">
+      <a href="#home" className="header__logo">
+        <img
+          src={NiceGadgetsLogo}
+          alt="logo"
+        />
+      </a>
+      <nav className="nav">
+        <ul className="nav__list">
+          <li className="nav__item">
+            <NavLink
+              to="/"
+              className={({ isActive }) =>
+                cn('nav__link', { 'is-active': isActive })
+              }
+            >
+              Home
+            </NavLink>
```

Comment:
It's better to make a separate component for navlink, `<MenuLink />` for example, and use in throughout the navigation.

Reply:
Yep, I also thought of that 
I`ll add it on next commit